### DISCUSSION TITLE
Static Map Edition?

### DISCUSSION THREAD

**Discussion by Bane000** [Nro001/Nro-Galaxy-Editor#3] [Q&A]
Hey, first of all great work, especially the interface looks quite useful :)
I wanted to ask if you plan to tweak the features of the editor in the future to give it the ability to work on static maps.
(Essentially outside of Save files being able to edit static galaxies for mods etc.)

**Comment by Nro001** [comment]
I probably wouldn't be able to work upon static maps, sorry. It's going to be an entirely new system rather than just tweaking my editor to work on it. I think I was only able to make this since I kept modifying the save files before, so I have no knowledge on editing static maps.
